aten/src/ATen/native/native_functions.yaml

```diff
@@ -4942,6 +4942,18 @@
   dispatch:
     CPU, CUDA: sign_out
 
+- func: signbit(Tensor self, *) -> Tensor
+  use_c10_dispatcher: full
+  variants: function, method
+
+- func: signbit_(Tensor(a!) self) -> Tensor(a!)
```

Comment:
Probably don't need an inplace variant for this function since it would only work on bool tensors.

Reply:
Adressed, the inplace variant has been deleted.